firefox_ui_harness/runtests.py

```diff
@@ -16,7 +16,7 @@
 
 class ReleaseTestRunner(BaseMarionetteTestRunner):
     extra_prefs = {
-        "browser.tabs.remote.autostart": True,
+        "browser.tabs.remote.autostart": False,
```

Comment:
Not until end of next week. Maybe one of the upcoming e10s fixes might solve it until then.